modules/cudf/src/series/string.ts

```diff
@@ -468,8 +509,9 @@ export class StringSeries extends Series<Utf8String> {
    * [...a.getJSONObject("$.foo").map(JSON.parse)] // object [{ bar: 'baz' }, { baz: 'bar' }]
    * ```
    */
-  getJSONObject(jsonPath: string, memoryResource?: MemoryResource): Series<Utf8String> {
-    return this.__construct(this._col.getJSONObject(jsonPath, memoryResource));
+  public static getJSONObject(jsonPath: string,
+                              memoryResource?: MemoryResource): Series<Utf8String> {
+    return Series.new(this.getJSONObject(jsonPath, memoryResource));
```

Comment:
This method is static now?

Reply:
That's weird. Typo got committed? I will revert that signature completely sigh.